examples/email-mcp/src/prompts.ts

```diff
@@ -0,0 +1,24 @@
+import { McpServer } from '@modelcontextprotocol/sdk/server';
+
+export function setupServerPrompts(server: McpServer) {
+  server.prompt(
+    'introduction',
+    'Learn about the Email MCP and how to use it',
+    () => ({
+      messages: [
+        {
+          role: 'assistant',
+          content: {
+            type: 'text',
+            text: `Welcome to the Email MCP.
+- Use "send_email" to send internal emails (verified recipients only).
+- Use "list_emails" to browse stored inbound emails.
+- Use "get_email" to fetch a specific stored email.
+
+Only internal routing is supported.`,
```

Comment:
but receiving publci emails is allowed. - you should give information to the AI of what public email people can email in order for the MCP tool to receive it.